### PR TITLE
♻️ Refactor Issue Templates

### PR DESCRIPTION
**Pull request by Gary-H9**
This PR refactors the Issue Templates to be more user friendly. This applies to bug reports, story templates and feature requests.

## Before
Screenshot 2023-08-22 at 14 20 50

## After
Screenshot 2023-08-22 at 14 20 19

[Example](https://github.com/ministryofjustice/nvvs-devops/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml&title=%F0%9F%90%9B+%3Ctitle%3E).

Technically this feature is in [Beta](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) but its in use elsewhere in the organisation and can easily be reverted if required in future.

### CODE REVIEW

**Review by julialawrence — DISMISSED** [review]
LGTM

**Review by bagg3rs — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug-template.md [hunk 1]
```diff
@@ -1,27 +0,0 @@
----
```
Comment (julialawrence) [inline .github/ISSUE_TEMPLATE/bug-template.md:1]: Can we readd the existing templates with zenhub prefix so people can still use markdown if they'd like and that they can be used on zenhub for those who like to create issues there please?

**Review by julialawrence — CHANGES_REQUESTED** [review]
Love the new templates but we need to keep the old ones too.

**Review by julialawrence — APPROVED** [review]
LGTM

**Review by jacobwoffenden — APPROVED** [review]

**Pull request merged**